### PR TITLE
Webinar Landing page

### LINKED ISSUE #587: Webinar page: Marketers Guide to Modern Web Apps
The page will be used as a landing page for a new campaign on LinkedIn and will explain what to expect and contain a signup form which links to Mailchimp

- [x] Create design/layout
- [x] Finalize content
- [x] Add signup form (#585)

### PR DESCRIPTION
**Pull request by pichfl**
Closes #587 

[Rendered Demo](https://deploy-preview-588--objective-northcutt-37494c.netlify.com/webinars/pwa/)

### CODE REVIEW

File: config/routes-map.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ module.exports = function() {
     '/talks': { component: 'PageTalks', title: 'Talks', bundle: { asset: '/talks.js', module: '__talks__' } },
     '/why-simplabs': { component: 'PageWhySimplabs', title: 'Why simplabs' },
     '/work': { component: 'PageWork', title: 'Work' },
+    '/webinars/pwa': { component: 'PageLandingPwaWebinar', title: 'Webinar: Progressive Web Applications' },
```
Comment (marcoow) [inline config/routes-map.js]: as this is not about PWAs only, we should use a different URL, e.g. `/webinars/modern-web` or so.

**Comment by ghislaineguerin** [comment]
@marcoow the rebase from improved-shapes here is intentional, once that is merged we are going to fix this branch.

**Comment by marcoow** [comment]
Regarding these icons, I think they are cool and we should not change them for this PR but in general I have the feeling they don't fit in perfectly with our overall design language? As they have rounded corners etc. to me they don't seem to integrate well with our shapes, our logo etc. that are all based on triangles and have a pretty sharp and hard look. I feel something like http://www.species-in-pieces.com/# (not necessarily 3D but something that uses polygons) would be a better foundation for visual elements maybe and would it in great with the shapes, logo etc.?

**Comment by ghislaineguerin** [comment]
@marcoow Yes, I agree we need to further develop the shape theme beyond the shapes themselves. The webinar page at the time had to be released quickly so we couldn't create special assets.

**Comment by ghislaineguerin** [comment]
@pichfl percy looked fine. I approved the changes

**Review by ghislaineguerin — APPROVED** [review]

**Pull request merged**